packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -1,3 +1,28 @@
+const deepCopyObj = (obj) => JSON.parse(JSON.stringify(obj))
```

Comment:
BTW: This will fail with an error if the `obj` has a cyclic reference too 😉

Reply:
True, cyclic reference Not my best param 😆

Reply:
i totally removed this function